packages/host/app/components/card-catalog-modal.gts

```diff
@@ -54,30 +63,84 @@ export default class CardCatalogModal extends Component<Signature> {
                   data-test-create-new
                 >Create New</Button>
               {{/if}}
+              <div class='card-catalog__results-length'>{{if
+                  this.currentRequest.search.instances.length
+                  this.currentRequest.search.instances.length
+                  '0'
+                }}
+                results</div>
```

Comment:
There can also be only 1 result and we should show the singular form of "results" in that case

Reply:
Also I think "results" shouldn't be shown without a number in front of it, so either "x results" or nothing. If I understand correctly currently in this code there will be a moment where only "results" will be shown when there is no number yet?